lib/client/browser.js

```diff
@@ -219,6 +219,16 @@ require("./discovery.js");
 					});
 
 					QUnit.log(function(details) {
+						if (config.logAllAssersions) {
+							const result = {
+								description: details.message,
+								suite: details.module && [qunitHtmlFile, details.module, details.name, details.message] || [],
+								success: details.result,
+								log: testResult.errors || [],
+								time: new Date().getTime() - timer
+							};
+							karma.result(result);
```

Comment:
Shouldn't this be done after the code below (Line 232ff)?
Otherwise `testResult.success` and `testResult.errors` are not filled (see lines 248/249).

Reply:
I've changed accordingly, integration tests will be added soon.